projects/aca-content/src/lib/components/favorite-libraries/favorite-libraries.component.ts

```diff
@@ -47,6 +49,10 @@ export class FavoriteLibrariesComponent extends PageComponent implements OnInit
   list: FavoritePaging;
   isSmallScreen = false;
   columns: DocumentListPresetRef[] = [];
+  searchVisibility = false;
```

Comment:
can be moved to the base `PageComponent` to avoid copy-pasting in each implementation